base/system/diskpart/diskpart_msg.mc

```diff
@@ -7219,84 +7236,88 @@ Example:
     SET ID=ebd0a0a2-b9e5-4433-87c0-68b6b72699c7
 .
 Language=Romanian
-    Changes the partition type field for the partition with focus.
+    Schimbă registrul tipului partiției pentru partiția în cauză.
 
-Syntax:  SET ID={<BYTE> | <GUID>} [OVERRIDE] [NOERR]
+Sintaxă:  SET ID={<BYTE> | <GUID>} [OVERRIDE] [NOERR]
 
     ID={<BYTE> | <GUID>}
 
-                Specifies the new partition type.
-                For master boot record (MBR) disks, you can specify a partition
-                type byte, in hexadecimal form, for the partition. Any
-                partition type byte can be specified with this parameter except
-                for type 0x42 (LDM partition). Note that the leading '0x' is 
-                omitted when specifying the hexadecimal partition type.
-
-                For GUID partition table (GPT) disks you can specify a
-                partition type GUID for the partition. Recognized GUIDs
-                include:
-
-                    EFI System partition:
+                Specifică noul tip al partiției.
+                Pentru discurile cu Registrul principal de inițializare (MBR),
+                puteți speciifica puteți specifica un octet, în format
+                hexazecimal, pentru un tip de partiție. Orice octet care
+                descrie tipul de partiție poate fi specificată cu acest
+                parametru, exceptând pentru tipul 0x42 (partiție LDM). Rețineți
+                că primul '0x' este omis când se specifică partiții de tip
+                hexazecimal.
```

Comment:
This is wrong. There are no partition of hexadecimal type. There is only type of partition that gets specified in hexadecimal form. Please correct this.

Reply:
This has not been corrected yet. The correct is something like "când este specificată în hexazecimal un tip de partiție".